src/components/backend-ai-agent-list.ts

```diff
@@ -1044,14 +1115,55 @@ export default class BackendAIAgentList extends BackendAIPage {
         <div slot="footer" class="horizontal end-justified flex layout">
         <mwc-button
           unelevated
-          id="update-agent-setting-button"
           icon="check"
           label="${_t('button.Update')}"
           @click="${() => this._modifyAgentSetting()}"></mwc-button>
         </div>
       </backend-ai-dialog>
     `;
   }
+
+  render() {
+    // language=HTML
+    return html`
+      <div class="list-wrapper">
+        <vaadin-grid class="${this.condition}" theme="row-stripes column-borders compact" aria-label="Job list"
+                    .items="${this.agents}">
+          <vaadin-grid-column width="30px" flex-grow="0" header="#" text-align="center"
+                              .renderer="${this._indexRenderer}"></vaadin-grid-column>
+          <vaadin-grid-column resizable width="100px" header="${_t('agent.Endpoint')}"
+                              .renderer="${this._boundEndpointRenderer}">
+          </vaadin-grid-column>
+          <vaadin-grid-column auto-width resizable header="${_t('agent.Region')}"
+                              .renderer="${this._boundRegionRenderer}">
+          </vaadin-grid-column>
+          <vaadin-grid-sort-column auto-width flex-grow="0" resizable path="architecture" header="${_t('agent.Architecture')}">
+          </vaadin-grid-sort-column>
+          <vaadin-grid-column resizable auto-width flex-grow="0" header="${_t('agent.Starts')}" .renderer="${this._boundContactDateRenderer}">
+          </vaadin-grid-column>
+          <vaadin-grid-column resizable width="160px" header="${_t('agent.Allocation')}"
+                              .renderer="${this._boundResourceRenderer}">
+          </vaadin-grid-column>
+          <vaadin-grid-column resizable width="120px" header="${_t('agent.Utilization')}"
```

Comment:
In the 14-inch monitor, it looks like this. So it's better to widen the utilization column. 
<img width="1246" alt="image" src="https://user-images.githubusercontent.com/28584164/186073452-df2c4cc9-c7f1-4526-9714-6c10b86500ea.png">

Reply:
Okay, let's make it 10px wider than now.